src/component/slick/MultipleSlick.jsx

```diff
@@ -45,7 +49,14 @@ const MultipleSlick = forwardRef((props, ref) => {
                 <Slider ref={ref} {...settings}>
                     {slides.map((slide, index) => (
                         <div key={index}
-                        ><CardFollow text1={slide.text1} text2={slide.text2} img={slide.img} className={slide.className} /></div>
+                        ><CardFollow
+                                address='tu'
+                                text1={slide.text1}
+                                text2={slide.text2}
+                                img={slide.img}
+                                className={slide.className}
+                                typeSocial={slide.typeSocial}
+                            /></div>
```

Comment:
e nên có thêm tham số trạng thái card disasble | not yet followed| followed để hiển thị UI , thêm sự kiện onClick button ( (cái này mình sẽ gọi tời các function khác nhau). như e viết hiện tại chỉ có thể ap dụng cho 3 loại giả sử sau này có thêm 2 loại card nữa e lại phải vào sửa code tiếp